components/builder-depot/src/doctor.rs

```diff
@@ -154,7 +153,6 @@ impl<'a> Doctor<'a> {
 
     fn run(mut self) -> Result<Report> {
         try!(self.init_fs());
-        try!(self.truncate_datastore(&self.depot.datastore));
```

Comment:
I believe this functionality is still needed to clear the DB (ignore if it got moved somewhere else).